Doc/library/_interpreters.rst

```diff
@@ -0,0 +1,69 @@
+:mod:`_interpreters` --- Low-level interpreters API
```

Comment:
Good point.  For the most part I was following the precedent of the _thread module.  However, in this case the module is relatively unstable still and there is no "interpreters" module yet.  So a warning makes sense.